application/src/main/java/org/thingsboard/server/actors/device/DeviceActorMessageProcessor.java

```diff
@@ -68,6 +68,7 @@
 import org.thingsboard.server.common.msg.rule.engine.DeviceNameOrTypeUpdateMsg;
 import org.thingsboard.server.common.msg.timeout.DeviceActorServerSideRpcTimeoutMsg;
 import org.thingsboard.server.common.util.KvProtoUtil;
+import org.thingsboard.server.gen.transport.TransportProtos;
```

Comment:
please change the import to import only SessionCloseReason:

```suggestion
import org.thingsboard.server.gen.transport.TransportProtos.SessionCloseReason;
```